src/org/omegat/filters3/xml/xliff/EditXLIFFOptionsDialog.java

```diff
@@ -214,7 +233,7 @@ private void okButtonActionPerformed(java.awt.event.ActionEvent evt) {// GEN-FIR
 
     private void cancelButtonActionPerformed(java.awt.event.ActionEvent evt) {// GEN-FIRST:event_cancelButtonActionPerformed
         doClose(RET_CANCEL);
-    }//GEN-LAST:event_cancelButtonActionPerformed
+    }// GEN-LAST:event_cancelButtonActionPerformed
```

Comment:
NetBeans might have removed this comment initially because its opening sibling was accidentally formatted to have a space after the `//` like this.

I think if the spaces following the `//` are removed, NetBeans will correctly recognize this block as generated code.